Début service de Notifications (NOTIFICATIONS)

feuloren opened the pull request:
Pour la future appli payutc pour iOS/Android 
Design by @mattgu74

Commit:
Début service de Notifications (NOTIFICATIONS)

Commit:
Merge branch 'master' of github.com:payutc/server

Conflicts:
	src/Payutc/Mapping/Services.php
	src/Payutc/Service/ADMINRIGHT.php

mattgu74 commented on src/Payutc/Bom/Notification.php:
```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Blocked.class
```
Non !

mattgu74 commented on src/Payutc/Bom/Notification.php:
```diff
@@ -0,0 +1,43 @@
+/**
+ * Blocked.class
+ * 
+ * Gestion des blocages
```
idem

mattgu74 commented on src/Payutc/Bom/Notification.php:
```diff
@@ -0,0 +1,43 @@
+ * Blocked.class
+ * 
+ * Gestion des blocages
+ * Table: tj_usr_fun_blocked_blo
```
idem

mattgu74 commented on src/Payutc/Bom/Notification.php:
```diff
@@ -0,0 +1,43 @@
+        $query = $qb->execute();
+        
+        if ($query->rowCount() == 0) {
+            Dbal::conn()->insert('t_notification_not',
```
Il y'a deux opérations, il faudrait faire une transaction !

mattgu74 commented on src/Payutc/Migrations/Version20140607212315.php:
```diff
@@ -0,0 +1,32 @@
+{
+    public function up(Schema $schema)
+    {
+        // Dans GCM le token peut faire jusqu'à 4ko ! Pour l'instant on met un varchar(256)
```
Moai, c'est pas la doc qui dit ça mais un mec sur un forum. Pour moi tout les token font moins de 255.

Commit:
Notifications : contrainte unique sur les champs

Commit:
Notifications : support du changement d'utilisateur

Dans le cas où un autre utilisateur se connecte à une appli qui a déjà un token

Commit:
Notifications : check utilisateur connecté et app
